### PR TITLE
ADEN-4240 | create line item with associated creative

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: src/Inventory/Api/LineItemCreativeAssociationService.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			'creativeSet' => true,
+			'creativeId' => $creativeId
+		];
+
+		try {
+			$user = Authenticator::getUser();
+
+			if ( empty($lineItemId) ) {
+				$response['success'] = false;
+				$response['$message'] = 'Line item ID is empty - unable to associate creative';
```
Comment (fraszczakszymon) [inline src/Inventory/Api/LineItemCreativeAssociationService.php]: Typo? s/$message/message

File: src/Inventory/Controller/LineItemController.php [hunk 2]
```diff
@@ -3,45 +3,49 @@
...
-class LineItemController extends Controller
-{
+class LineItemController extends Controller {
 	protected $lineItemService;
+	protected $lineItemCreativeAssociationService;
 
-	public function __construct(Application $app)
-	{
-		parent::__construct($app);
+	public function __construct( Application $app ) {
```
Comment (fraszczakszymon) [inline src/Inventory/Controller/LineItemController.php]: Do we really want to go with MW style guides? :P

Comment (bognix) [reply src/Inventory/Controller/LineItemController.php]: Don't know, those are my IDE style guides, I can revert

**Comment by fraszczakszymon** [comment]
LGTM 👍

**Comment by bulaszenko** [comment]
checked works good

**Pull request merged**